### ISSUE TITLE
Platypus: Missing main landmark

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/platypus#474]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the page is missing a main landmark.

---

https://dequeuniversity.com/rules/axe/4.3/landmark-one-main?application=axeAPI

---

**User impact:** Moderate
**Disabilities affected:** Blind, Deafblind, Mobility
**Requirement(s):** Best Practice
**WCAG Success Criteria:** N/A

**Comment by micpap25** [comment]
Hi, I believe all the main landmarks are present in the HTML; could you confirm this?
<img width="290" alt="image" src="https://user-images.githubusercontent.com/34077605/212339008-cf6b5df8-e45e-455b-8dbe-3d87ef5cd320.png">

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
